recipes/newrelic/apm/dotNet/windows/net-install.yml

```diff
@@ -47,12 +43,49 @@ install:
         - task: start
         - task: ensure_sites_started
 
+    verify_continue:
+      cmds:
+        - |
+          powershell -command '
+          Write-Host ""
+          Write-Host -ForegroundColor Yellow "================================================================================"
+          Write-Host -ForegroundColor Yellow "=                                                                              ="
+          Write-Host -ForegroundColor Yellow "=                                   Warning                                    ="
+          Write-Host -ForegroundColor Yellow "=                                                                              ="
+          Write-Host -ForegroundColor Yellow "=       This installation will automatically restart IIS upon completion.      ="
+          Write-Host -ForegroundColor Yellow "=                                                                              ="
+          Write-Host -ForegroundColor Yellow "================================================================================"
+          Write-Host ""
+          Write-Host "If you are hosting your .NET application differently then check out our other installation options:"
+          Write-Host "https://docs.newrelic.com/docs/agents/net-agent/installation/install-net-agent-windows/."
+          Write-Host ""
+          if (! $env:NEW_RELIC_ASSUME_YES) {
```

Comment:
I would have expected we use go-task var syntax here like this:
if (! {{.NEW_RELIC_ASSUME_YES}} ) {

Reply:
I was assuming it was an env var. I will update things to capture the go-task value and handle it.